test/factories/sheltered.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
```

Comment:
we are using fixtures do deal with database 
`test/fixtures/*`

I am ok to change to use factory bot, but we will need to change other fixtures as well and tests